contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -56,7 +57,29 @@ var ChannelList  = BaseViews.BaseEditableListView.extend({
 		$(".disable-on-edit").prop("disabled", edit_mode_on);
 		$(".disable-on-edit").css("cursor", (edit_mode_on) ? "not-allowed" : "pointer");
 		$(".invisible-on-edit").css('visibility', (edit_mode_on)?'hidden' : 'visible');
+		(edit_mode_on)? $(".new_channel_button").addClass("disabled") : $(".new_channel_button").removeClass("disabled");
+		$(".new_channel_button").html( (edit_mode_on)? '<i>Edit in progress...</i>' : "ADD CHANNEL");
+		$(".new_channel_button").prop('title', (edit_mode_on)? 'Cannot create a new channel while another channel is being edited.' : "Create a new channel");
 	},
+	update_dropdown:function(channel, deleted){
```

Comment:
https://trello.com/c/s99JaY11/233-channel-drop-down-feature-does-not-update-when-you-add-new-channels